### PR TITLE
Cleanup internal org

### PR DESCRIPTION
**Pull request by cournape**
Reorganize and rationalize a bit the package API:

* purely SAT logic is now in `simplesat.sat`
* rest of the code stays in `simplesat` itself
* removed old solver code, and move 0install-based algorithm in old directory

@jvkersch

### CODE REVIEW

File: simplesat/add_clause_script.py [hunk 1]
```diff
@@ -1,5 +0,0 @@
-from simplesat.api import Solver, Clause
```
Comment (jvkersch) [inline simplesat/add_clause_script.py]: I don't even remember where this came from ;-)

**Comment by jvkersch** [comment]
LGTM.

Travis builds weren't running anymore since the repo was open-sourced. I enabled builds on the open-source Travis infrastructure and kicked off a test run.

**Comment by jvkersch** [comment]
... which fails because of the dependency on enstaller.

File: simplesat/add_clause_script.py (hunk 1 above)
Comment (cournape) [inline simplesat/add_clause_script.py]: Let's pretend it never happened :)

**Comment by cournape** [comment]
I put some commits I should not have, recreating a new PR.

**Pull request closed without merging**